src/EVEMon/SkillPlanner/PlanWindow.Designer.cs

```diff
@@ -330,7 +342,7 @@ private void InitializeComponent()
             this.planEditor.Dock = System.Windows.Forms.DockStyle.Fill;
             this.planEditor.Location = new System.Drawing.Point(0, 0);
             this.planEditor.Name = "planEditor";
-            this.planEditor.Size = new System.Drawing.Size(816, 480);
+            this.planEditor.Size = new System.Drawing.Size(935, 480);
```

Comment:
Did you make the default plan window bigger to fit the new button? This is fine, but wanted to make sure this parameter change was not due to a VS auto-resize for DPI or other reasons.

Reply:
Yes I increased the size myself to accommodate the button.

Reply:
Now that 4.0.3 is out I think I can take this and look at it for the next release. Thanks for the great work!